tests/optim_test.py

```diff
@@ -15,6 +15,7 @@
 # Lint as: python3
 """Tests for flax.optim."""
 
+import pytest
```

Comment:
I am not sure if there are any guidelines flax follows on the import order, but I would put it as the last import (i.e after flax.optim.weight_norm...). 

Ideally we should be following: https://www.python.org/dev/peps/pep-0008/#imports or conventions adopted by Flax. 

@avital wdyt?

Reply:
I didn't know PEP-8 said anything about imports, that's neat thanks!

Reply:
Agree that it'd be good to pick a convention and automate it, but perhaps developers will find it annoying to have to re-commit if files get re-formatted.

I've used this on previous projects: https://pre-commit.com/. This simplifies adding git pre-commit hooks (e.g., formatting, linting).

With regard to sorting imports, I've used `isort`.

Reply:
Thanks for the suggestions Daniel. I plan to configure a linter soon for Flax. Will take this into consideration.

Reply:
@mohitreddy1996 I'll create a new issue so we can move the conversation there

Reply:
Yeah I do think standardizing import order helps if it's streamlined but let's not worry about that right now before we have a linter in place.